content/news/playtest-20180101.md

```diff
@@ -0,0 +1,63 @@
+We are excited to welcome the new year with a new [OpenRA playtest](/downloads/)!
+More than 700 commits from 42 contributors were merged over the last five months, with each commit helping to fix bugs, improve code quality, or add new features that improve the single- and multi-player experience.
+
+This playtest is a timely opportunity to get community participation, as we celebrate a massive record of games played. At the end of November we passed 500,000 completed multiplayer games since August 2014, when we started recording online server statistics. This is a huge accomplishment for our community, congratulations!
+
+The main focus of this development cycle was modernizing core parts of the engine and game code to support the work-in-progress Tiberian Sun mod and other community projects. These shouldn't have visibly changed our released mods, but big changes always have a risk of introducing unexpected bugs or side-effects. Please note that changes to the way hotkeys are managed means you will need to redefine any changes you made in the Input settings.
+
+The changes weren't all behind the scenes; an overhaul of the multiplayer UI added long-requested features:
+
+ * The server browser shows the name/faction/team of connected players
+ * The server browser is now also available as a tab in the server lobby
+ * The main menu and server browser will notify you if a new release is available
+ * The Global Chat has been removed from the server browser and lobby
+ * A chat dialog has been added to the in-game menu
+
+<div style="text-align:center" markdown="1">
+<img src="/images/news/20180101-serverplayers.png" width="600" alt="Server player list">
+
+The new server player list is one of several improvements to the multiplayer UI
+</div>
+
+Of course, we realize that the vast majority of OpenRA players do not play online. We haven't forgotten about you either!
+Improvements for the singleplayer experience include:
+
+ * The full Harkonnen campaign in D2K
+ * Refinements and improvements to several campaign missions in Red Alert and Tiberian Dawn
+ * A fix for the AI cheating when it targets superweapons and support powers
```

Comment:
`targets` might be confusing here. At first I thought the AI is targeting the powers. We could use `fires` instead.